run/README.md

```diff
@@ -60,21 +60,26 @@ For more Cloud Run samples beyond Python, see the main list in the [Cloud Run Sa
     Injecting your service account key:
 
     ```
-    export SA_KEY_NAME=my-key-name-123
+    export SA_KEY_PATH=/path/to/service/account/my-key-name-123.json
+
     PORT=8080 && docker run --rm \
         -p 8080:${PORT} -e PORT=${PORT} \
-        -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/keys/${SA_KEY_NAME}.json \
-        -v $GOOGLE_APPLICATION_CREDENTIALS:/tmp/keys/${SA_KEY_NAME}.json:ro \
+        -e GOOGLE_APPLICATION_CREDENTIALS=$SA_KEY_PATH \
+        -v $GOOGLE_APPLICATION_CREDENTIALS:$SA_KEY_PATH:ro \
```

Comment:
Possibly this code block needs more comments in general. Let me break down what I had happening originally:

1. SA_KEY_NAME is set to the name of the service account key file as we want it to exist inside the container
2. As a convention, we've decided the /tmp/keys/ directory is where the key will be found inside the container
3. The GOOGLE_APPLICATION_CREDENTIALS env var inside the container is set to look for that file
4. The volume mount is using the $GOOGLE_APPLICATION_CREDENTIALS variable on the host machine.

The result it the local machine's $GOOGLE_APPLICATION_CREDENTIALS is being made available in a central, generic location inside the container.

When we adapt that to the refactor here, the $SA_KEY_PATH variable is a bit odd without a comment, because the path used there is arbitrarily selected as a location inside the container. It's something we can dictate and does not need to relate to the local host.

Hopefully that wasn't too confusing. Based on that explanation, we either:

1. Revert to the original and add sufficient text to clarify.
2. Keep the new version and add sufficient text to clarify.